views/about.handlebars

```diff
@@ -0,0 +1,88 @@
+<main role="main" xmlns="http://www.w3.org/1999/html">
+    <div class="container center-content about-container-padding">
+        <!--        <img src="/images/logo.jpg" alt="Botomir Logo">--> <!-- Use Botomir logo when ready -->
+        <h1 class="about-header-padding">About Botomir</h1>
+        <p class="align-text-left about-description-padding">
+            Botomir is a Discord Bot written in Javascript. They were born on November 25, 2020, and was a happy
+            accident more than a planned project. Botomir was created because he is a remedy to loneliness. It can be
+            just you and Botomir, they won't tell.
+
+            Botomir loves to eat chicken and waffles, washing it down with a Four loco, and cookies for dessert. Their
+            favourite song is the windows startup sound. His hobbies include going on long scrolls through Wikipedia,
```

Comment:
Gendering of botomir, everywhere else gender neutral phrases are used, should this `his` be changed to `their`?

Reply:
Yes, will change

Reply:
Fixed